src/Services/EstimateService.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class EstimateService {
+    public static function convertEstimate(?int $estimate): string|null
```

Comment:
The `?` before `int` suggests you want to accept `null` as an argument. I don't see why you'd ever want that, though. Why is it there?

Reply:
Some database entries come through with a null value, so we added this in - it seems to break the code if we don't allow for null values!

Reply:
I think the best way to deal with that is to check for a null value before using the `convertEstimate` method at all, rather than trying to use it on a null value!